docs/team/abnermtj.md

```diff
@@ -1,29 +1,56 @@
-
 # Tolentino Abner Jr Morales Project Portfolio Page 
 
 ## Overview
-CheatLogs is a free and open-source cheatsheet manager with blazing fast organization, editing and searching via a command-line interface (CLI) and graphical user interface (GUI) . 
+CheatLogs is a free and open-source cheatsheet manager with blazing fast organization, editing, and searching via a command-line interface (CLI) and graphical user interface (GUI). 
 
 Written in Java and has about 5000 Lines of Code (LoC).
 
-## Summary of Contributions
+### Summary of Contributions
+#### Code Contributed
+
+[Reference for code ](https://nus-cs2113-ay2021s1.github.io/tp-dashboard/#breakdown=true&search=abnermtj&sort=groupTitle&sortWithin=title&since=2020-09-27&timeframe=commit&mergegroup=&groupSelect=groupByRepos&checkedFileTypes=docs~functional-code~test-code~other&tabOpen=true&tabType=authorship&tabAuthor=abnermtj&tabRepo=AY2021S1-CS2113T-W11-3%2Ftp%5Bmaster%5D&authorshipIsMergeGroup=false&authorshipFileTypes=docs~functional-code~test-code)
+
+---
+#### Features/Enhancements Implemented
+I was in charge of the classes: **Command** , **Parser**, **UserSesssion**, **UI** , **ConsoleColors**, **ClearCommand**, **DeleteCommand**, **EditCommand**, **ExitCommand**, **HelpCommand**
+
+I will go through mainly the **Parser**, **Command**, and **UserSession** and **ConsoleColors** classes to make this not too long.
+##### Parser class
+This class parses the user input to construct a Command object that can be executed at a later time. It performs input validation to prevent exceptions. This class also was improved to prompt for the user input in the case that he missed some of the required flags.	This was particularly tricky because there are so many possible ways input can go wrong and had to be constantly adjusted throughout the project.
```

Comment:
You can separate the challenges you faced in a separate section. For example:

*Highlights*
    ... your hightlights